setup.cfg

```diff
@@ -60,3 +69,6 @@ max-line-length = 99
 
 [mypy]
 ignore_missing_imports = True
+
+[tool:pytest]
+asyncio_mode = auto
```

Comment:
according to my understanding of https://github.com/openlawlibrary/pygls/blob/4772250f8ede25aee8e457dae5bca394aeccdd2f/.editorconfig#L7 there should be a new line at end of file here